apps/src/lib/util/AnalyticsConstants.js

```diff
@@ -36,7 +36,6 @@ const EVENTS = {
   COUNTRY_SELECTED: 'User Selects Country',
   ZIP_CODE_ENTERED: 'Valid Zip Code Entered',
   SCHOOL_SEARCH_INPUT_ENTERED: 'School Search Input Entered',
-  SCHOOL_LIST_OPENED: 'School List Opened',
```

Comment:
This is a followup from a conversation here: https://docs.google.com/document/d/1Ulaxj3K6AMNOOjw_244LNOV5ypuHJMjGpUItXpZuC4I/edit?disco=AAABJx8KQyg